docs/reference-guide/storage-modes/in-memory-analytical.md

```diff
@@ -0,0 +1,37 @@
+---
+id: in-memory-analytical
+title: In-memory analytical
+sidebar_label: In-memory analytical
+slug: /reference-guide/storage-modes/in-memory-analytical
+---
+
+`IN_MEMORY_ANALYTICAL` mode offers no ACID guarantees besides manually created snapshots. There are no `WAL` files created or periodic snapshots. Users **can** create a snapshot manually. 
+
+In `IN_MEMORY_TRANSACTIONAL` mode, during updates, there are [delta](../../under-the-hood/storage.md#delta-memory-layout) objects created for each change of data. Such a concept is the backbone for 
+Memgraph to provide atomicity, consistency, isolation, and durability - ACID. Using `deltas`, Memgraph creates write-ahead-logs (durability), provides isolation and consistency, and atomicity (either everything is executed or nothing). 
+
+One big overhead with `deltas` is that they require a lot of memory (104B per change) especially when there are a lot of changes (import, i.e. `LOAD CSV` query). `IN_MEMORY_ANALYTICAL` disables the creation of `deltas` in order to provide faster import with lower memory consumption. Also, such a mode creates a speedup of import and reduces memory usage drastically - up to 6 times.
+
+
+## Transactions
+
+As there are no ACID guarantees, other transactions can see changes ongoing transactions are doing. Also, a transaction can see the changes it is doing.
+
+## WAL
+
+As mentioned, there are no write-ahead logs created during the `IN_MEMORY_ANALYTICAL` mode. When switching back to `IN_MEMORY_TRANSACTIONAL` it is recommended to create a snapshot manually. After switching, `WAL` is created
+if enabled with the config file.
```

Comment:
```suggestion
As mentioned, no write-ahead logs are created in the `IN_MEMORY_ANALYTICAL` mode. When switching back to the `IN_MEMORY_TRANSACTIONAL` mode it is recommended to create a snapshot manually with `CREATE SNAPSHOT;` Cypher query. Once Memgraph switches to the `IN_MEMORY_TRANSACTIONAL` mode, it will create a WAL if not otherwise instructed by the config file.
```